companion/src/storage/appdata.cpp

```diff
@@ -621,6 +619,8 @@ void AppData::init()
     getset( _theme,           "theme"                   ,1  );
     getset( _warningId,       "warningId"               ,0  );
     getset( _simuLastProfId,  "simuLastProfId"          ,-1 );
+    getset( _firmwareBranch,  "firmwareBranch"          , BRANCH_RELEASE_STABLE );
+    getset( _companionBranch, "companionBranch"         , BRANCH_RELEASE_STABLE );
```

Comment:
`firmwareBranch_init()` and `companionBranch_init()` should work also... avoids re-defining the default values.

Reply:
ah yes, CLion proposes them in the completion ;)